twitter/searcher.py

```diff
@@ -120,19 +125,61 @@ def twitter_created_at_to_utc(created_at: str):
 
 # TODO: hook up to s3 bucket
 def log_last_processed_id(last_processed_id: int, last_processed_time_stamp: str):
-    msg = {"last_processed_id": last_processed_id, "last_processed_time_stamp": last_processed_time_stamp}
+    msg = {
+        "last_processed_id": last_processed_id,
+        "last_processed_time_stamp": last_processed_time_stamp,
+    }
     with open(TWITTER_LAST_RUN_FILENAME, "w") as f:
         f.write(json.dumps(msg))
-    # s3_client = boto_client("s3")
-    # response = s3_client.upload_file(TWITTER_LAST_RUN_FILENAME, TWITTER_LAST_RUN_BUCKET, TWITTER_LAST_RUN_FILENAME)
-    # return response
 
 
-def get_since_id_from_file():
+def get_since_id_from_file(self):
```

Comment:
I think the self param can be removed here

Reply:
Actually, I think these non-class functions can just be removed wholesale